### ISSUE TITLE
Cobalt serve won't rebuild site when the destination directory is "./"

### ISSUE THREAD

**Issue by natjms** [cobalt-org/cobalt.rs#1020]
I have my website set up such that all of the rendered HTML files and assets are outputted into the top level directory (in the same folder as my `_cobalt.yml`). All of my source files are stored in a folder called `_src` within that same directory. So, in my configuration file, I have

```
source: _src
dest: .
```

I'm not entirely sure why I'm doing this, but I set it up a while ago and haven't gotten around to changing it. With this setup, running `cobalt serve` won't do anything besides render your source files once and then serve them on an HTTP server. If I change one of the files in the `_src` directory, the change won't get logged in the console and if I refresh the browser, nothing gets updated. I thought this might have something to do with preventing an infinite loop of building and rebuilding, but I suppose that probably wouldn't happen unless, for whatever reason, you wanted to output files into your source directory. If it is necessary, it'd be worth documenting it.

I've tested this on Cobalt 0.17.5, using Ubuntu 20.04.3 LTS.

**Comment by epage** [comment]
Thanks for reporting this!

I probably won't get to this for a couple of days but wanted to at least acknowledge that I saw it.

**Comment by epage** [comment]
So this is already fixed in `master`, in a round about way.  I guess I've lost track of making releases.

In `master`, `cobalt serve` will now use a temp directory as the destination.  `cobalt serve` builds the site in a different way (e.g. changing `site.base_url`) and we didn't want to accidentally leave those files around and corrupt someone's actual site.

Once the fix for your other issue is merged, I'll do a release.
